src/components/DevDummyLogin/index.js

```diff
@@ -140,6 +153,8 @@ const DevDummyLogin = ({resetClickTime = () => {}}) => {
   };
 
   const dummyLogin = (appUserId) => {
+    if (!appUserId) return;
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good addition of a guard clause to prevent further execution if `appUserId` is not provided.



<!-- This is an auto-generated comment by OSS CodeRabbit -->